### PR TITLE
use getUrlPath instead of collective.slug in automatic retweets

### PR DESCRIPTION
**Pull request by EslamHiko**
This Pr is solving issue: https://github.com/opencollective/opencollective/issues/1270

I need help with writing tests.

### CODE REVIEW

File: cron/10mn/milestones.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const init = () => {
     },
     limit: 30,
     group: ['CollectiveId', 'collective.id'],
-    include: [{ model: models.Collective, as: 'collective' }],
+    include: [{ model: models.Collective, where: { type: { [Op.ne]: collectiveTypes.EVENT } }, as: 'collective' }],
```
Comment (Betree) [inline cron/10mn/milestones.js]: Where is `collectiveTypes` imported?

**Comment by znarf** [comment]
@EslamHiko can you maybe push with Git to our main repository so that CI can be triggered?

**Comment by znarf** [comment]
- Can you check lint?
- Can you rebase and push the result?

**Comment by Betree** [comment]
Because these kind of issues take some time to test, it's a great practice to add screen(shot/cast)s

**Pull request merged**